app.js

```diff
@@ -1,14 +1,34 @@
 
 document.getElementById('submit').addEventListener('click', function() {
-let name = document.getElementById("name").value;
-let dob = document.getElementById("dob").value;
-let email = document.getElementById("email").value;
-let phone = document.getElementById("phone").value;
-let city = document.getElementById("city").value;
-let institute = document.getElementById("institute").value;
+let name = document.getElementById("name");
+let dob = document.getElementById("dob");
+let email = document.getElementById("email");
+let phone = document.getElementById("phone");
+let city = document.getElementById("city");
+let institute = document.getElementById("institute");
 var gen = document.getElementsByName("gender");
 var qual = document.getElementsByName("c");
 
+
+    if (name.value == 0) {
+      window.alert("please enter your name");
+      return false;
+    }
+    else if(dob.value == 0) {
+        window.alert("please enter your date of birth");
+        return false;
+      }
+    else if(phone.value == 0) {
+        window.alert("please enter your contact number");
+        return false;
+      }
+    else if(email.value == 0) {
+        window.alert("please enter your emaili");
+        return false;
+      }
+    else{
+    
+ 
     for(let i = 0; i<gen.length; i++){

```

Comment:
by the time user reaches here, if he didn't enter valid values above, he might have seen 5-6 popups, this is a bad user experiance. 

update this as soon as you can

Reply:
popup comes one by one